<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileExtensionWidget</class>
 <widget class="QWidget" name="FileExtensionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QLabel" name="classLabel">
     <property name="text">
      <string>Associated class:</string>
     </property>
     <property name="buddy">
      <cstring>classComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="fileExtentionLineEdit"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="classComboBox"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="settingsCheckBox">
     <property name="text">
      <string>Configure class settings</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="fileExtentionLabel">
     <property name="text">
      <string>File extension:</string>
     </property>
     <property name="buddy">
      <cstring>fileExtentionLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="iconPathLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>117</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Icon file path:</string>
        </property>
        <property name="buddy">
         <cstring>iconPathLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="iconPathLineEdit"/>
      </item>
      <item row="0" column="3">
       <widget class="QToolButton" name="iconPathToolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="iconIndexLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>117</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>105</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Icon index:</string>
        </property>
        <property name="buddy">
         <cstring>iconIndexLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="iconIndexLineEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="actionIconLabel">
        <property name="text">
         <string>Actions:</string>
        </property>
        <property name="buddy">
         <cstring>iconTableWidget</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="3" colspan="2">
       <widget class="QTableWidget" name="iconTableWidget"/>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QPushButton" name="newPushButton">
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QPushButton" name="editPushButton">
        <property name="text">
         <string>Edit</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QPushButton" name="removePushButton">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="openAftercheckBox">
        <property name="text">
         <string>Confirm open after download</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QPushButton" name="setDefaultPushButton">
        <property name="text">
         <string>Set Default</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QCheckBox" name="alwaysCheckBox">
        <property name="text">
         <string>Always show extension</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QCheckBox" name="browseCheckBox">
        <property name="text">
         <string>Browse in some window</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="actionLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Action:</string>
      </property>
      <property name="buddy">
       <cstring>actionComboBox</cstring>
      </property>
     </widget>
     <widget class="QComboBox" name="actionComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentText">
       <string>Create</string>
      </property>
      <item>
       <property name="text">
        <string>Create</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Replace</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Update</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Delete</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>actionComboBox</tabstop>
  <tabstop>fileExtentionLineEdit</tabstop>
  <tabstop>classComboBox</tabstop>
  <tabstop>settingsCheckBox</tabstop>
  <tabstop>iconPathLineEdit</tabstop>
  <tabstop>iconPathToolButton</tabstop>
  <tabstop>iconIndexLineEdit</tabstop>
  <tabstop>iconTableWidget</tabstop>
  <tabstop>newPushButton</tabstop>
  <tabstop>editPushButton</tabstop>
  <tabstop>removePushButton</tabstop>
  <tabstop>setDefaultPushButton</tabstop>
  <tabstop>openAftercheckBox</tabstop>
  <tabstop>alwaysCheckBox</tabstop>
  <tabstop>browseCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>settingsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
